src/app/core/entity/default-datatype/default.datatype.ts

```diff
@@ -103,8 +86,8 @@ export class DefaultDatatype<EntityType = any, DBType = any> {
   async importMapFunction(
     val: any,
     schemaField: EntitySchemaField,
-    additional?: any,
-  ): Promise<EntityType | EntityType[]> {
+    additional?: string,
```

Comment:
as this base class gets extended by various implementation, `additional` can have different types here and cannot be forced to string always. Leave this as "any" here. In the EntityDatatype class, you should be able to change the parameter to have type "string" in that specific implementation.


```suggestion
    additional?: any,
```